lib/export/dwca/data.rb

```diff
@@ -2,6 +2,14 @@
 
 module Export::Dwca
 
+  # Columns we include in the Resource Relationship extension table that
+  # aren't DwC terms but have meaning for us and perhaps users of our
+  # archives.
+  LOCAL_RESOURCE_RELATIONSHIP_TERMS = {
+    'TW:Resource': 'https://sfg.taxonworks.org/dwc/terms/resourceRelationship/resource',
+    'TW:RelatedResource': 'https://sfg.taxonworks.org/dwc/terms/resourceRelationship/relatedResource',
```

Comment:
The biological associations extension currently includes two columns (resource and related resource, index 3 and 7 below) that aren't dwc terms. These are required to have a term field in the `meta.xml` file - I just made one up here, but we should decide on what we really want (since it should be permanent I guess).

<img width="1012" height="368" alt="Image" src="https://github.com/user-attachments/assets/7fb17d26-f0db-4c25-875c-d141f3290c54" />